data/json/snippets/graffiti.json

```diff
@@ -110,7 +145,21 @@
       { "id": "shelter_graffiti_32", "text": "GOVERMENT DID THIS TO US" },
       { "id": "shelter_graffiti_33", "text": "FUCK CHINA" },
       { "id": "shelter_graffiti_34", "text": "FUCK THE COMMUNISTS" },
-      { "id": "shelter_graffiti_35", "text": "Remember <given_name>" }
+      { "id": "shelter_graffiti_35", "text": "Remember <given_name>" },
+      {
+        "id": "shelter_graffiti_36",
+        "text": "Two homeless men were going at it while a zombie was eating a woman alive right next to them.  They even invited the zombie to join them."
+      },
+      {
+        "id": "shelter_graffiti_37",
+        "text": "A pregnant woman said that one of those undead fuckers fucked her in the fucking fuck fuck.  Fuck.  I had sloppy seconds.  It was worth it."
+      },
```

Comment:
Like a solid half of those reek of extremely poor middle school levels of humor. I don't think I've ever been less of a fan of a PR.